### PR TITLE
Epic 7 - Pavlo Yaremchuk

### PR DESCRIPTION
**Pull request by PavloYaremchuk**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/pavlo_yaremchuk/Epic7/kr.cpp [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+using namespace std;
+
+const int nomer = 990;
+
+double func(double a, double b)
+{
+    return a + b;
+}
+
+void createMatrix(double a, double matrix[10][10];)
```
Comment (fm2lpnu) [inline ai_12/pavlo_yaremchuk/Epic7/kr.cpp:15]: Task 1 (a,b) is not implemented according to requirements.
None of two options is implemented

File: ai_12/pavlo_yaremchuk/Epic7/kr.cpp [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+#include <cstdlib>
+#include <fstream>
+
+using namespace std;
+
+const int nomer = 990;
+
+double func(double a, double b)
+{
+    return a + b;
```
Comment (fm2lpnu) [inline ai_12/pavlo_yaremchuk/Epic7/kr.cpp:12]: Is not implemented according to requirement. 
Math function had to be taken from vns_practice works

File: ai_12/pavlo_yaremchuk/Epic7/kr.cpp [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    double q1, w1, e1, r1, t1, y1, u1, i1, o1, p1;
+    double a = func(q, q1);
+    double b = func(w, w1);
+    double d = func(e, e1);
+    double f = func(r, r1);
+    double g = func(t, t1);
+    double h = func(u, u1);
+    double j = func(i, i1);
+    double k = func(o, o1);
+    double l = func(p, p1);
```
Comment (fm2lpnu) [inline ai_12/pavlo_yaremchuk/Epic7/kr.cpp:132]: The code in `main` function  is nonsense

File: ai_12/pavlo_yaremchuk/Epic7/kr.cpp [hunk 4]
```diff
@@ -0,0 +1,139 @@
+#include <iostream>
+#include <ctime>
+#include <cstdlib>
+#include <fstream>
+
+using namespace std;
+
+const int nomer = 990;
```
Comment (fm2lpnu) [inline ai_12/pavlo_yaremchuk/Epic7/kr.cpp:8]: Lack of any comments that prove that work was done with understanding and used required valuation criterias

**Review by fm2lpnu — APPROVED** [review]

**Comment by fm2lpnu** [comment]
Updated PR:
https://github.com/artificial-intelligence-department/ai_programming_playground/pull/1384

**Pull request closed without merging**